ocs_ci/utility/openshift_dedicated.py

```diff
@@ -22,7 +22,12 @@ def login():
     Login to OCM client
     """
     token = openshift_dedicated["token"]
-    cmd = f"ocm login --token={token} --url=staging"
+    ms_env = config.ENV_DATA.get("ms_env_type", "")
```

Comment:
There could be `"staging"` instead of `""` to make it obvious that staging environment is the default one. It could be also part of [default config](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/conf/default_config.yaml) so that a correct value is returned when querying `config.ENV_DATA.get("ms_env_type")`.